### PR TITLE
8 user rename

### PR DESCRIPTION
**Pull request by Relentlesslba**
Moved proto building to protobuild. To build:
./protobuild/cargo run

Client:
Added set-group
Added rename-user

Updated readme

### CODE REVIEW

File: proto/manageusers.proto [hunk 1]
```diff
@@ -7,6 +7,13 @@ service ManageUsers {
   rpc ListAllUsers(EmptyParams) returns (stream User) {}
   rpc ListAllGroups(EmptyParams) returns (stream Group) {}
   rpc DeleteUser(WhichUser) returns (GenericError) {}
+  rpc RenameUser(RenamedUser) returns (GenericError) {}
+  rpc SetGroup(SetGroupParms) returns (GenericError) {}
```
Comment (ytitov) [inline proto/manageusers.proto]: All looks good except maybe rename GenericError to GenericResult?  GenericError implies that it is an error no matter what the response is.

Comment (Relentlesslba) [reply proto/manageusers.proto]: ok, renamed

**Comment by Relentlesslba** [comment]
gen proto script will generate all protos

**Comment by Relentlesslba** [comment]
btw, no IDE = no refractor tool

**Pull request merged**